policyengine_us/variables/gov/states/wi/tax/income/credits/earned_income/wi_earned_income_credit.py

```diff
@@ -19,7 +19,12 @@ class wi_earned_income_credit(Variable):
     def formula(tax_unit, period, parameters):
         p = parameters(period).gov.states.wi.tax.income.credits
         inv_income = tax_unit("eitc_relevant_investment_income", period)
-        ineligible = inv_income > p.earned_income.investment_income_limit
+        # In 2023 Wisconsin adopted the federal EITC investment income limit
+        if period.start.year >= 2023:
```

Comment:
Make this a bool parameter and then use the if statement to define the limit. Define eligibility (rather than ineligibility) outside the if statement